components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -626,6 +627,35 @@ class Airwallex internal constructor(
         )
     }
 
+    /**
+     * Checkout the payment by paymentType and session
+     *
+     * @param session a [AirwallexSession] used to present the Checkout flow, required.
+     * @param paymentType a [String] representing one of the redirect payment type names, required.
+     * @param additionalInfo a [Map] containing extra information needed for certain payment types, such as phone number, email, bank details, etc., optional.
+     * @param flow an [AirwallexPaymentRequestFlow], currently only supporting [AirwallexPaymentRequestFlow.IN_APP], optional.
+     * @param listener the callback for the checkout result.
+     */
+    @UiThread
+    fun checkout(
+        session: AirwallexSession,
+        paymentType: String,
```

Comment:
maybe calling `paymentMethodName` is more explicit, since it's from `name` fields of `https://www.airwallex.com/docs/api#/Payment_Acceptance/Config/_api_v1_pa_config_payment_method_types/get`